metabase/metabase
# Clip voronoi by X axis to prevent overlapping events click areas

Epic https://github.com/metabase/metabase/issues/20289

When chart dots are close to the X axis, they should not completely overlap event icons.

How to test:
- Create the chart as specified on the screenshot
- Make sure it's possible and easy to hover over the event icon

<img width="1534" alt="Screenshot 2022-03-23 at 18 42 27" src="https://user-images.githubusercontent.com/8542534/159739231-dae8e2c7-4771-4134-9958-dd5ba906842e.png">
<img width="1602" alt="Screenshot 2022-03-23 at 18 42 38" src="https://user-images.githubusercontent.com/8542534/159739266-9d2314b7-c271-4779-80ae-3f5f7433c6b8.png">
